constants/error-constants.js

```diff
@@ -21,6 +21,7 @@ const ERROR = {
         // submission action
         INVALID_SUBMISSION_ACTION: "Invalid submission action:",
         INVALID_SUBMISSION_ACTION_STATUS: "Invalid submission status for the action:",
+        INVALID_SUBMIT_ACTION: "The submit action should only be allowed if it meets both the user's role and the status of the data submission",
```

Comment:
Let's reword this to "Invalid submission action, user role and submission status requirements are not met."